### PR TITLE
Fix READMEs et al

### PR DESCRIPTION
**Pull request by awholegunch**
Correct copyright statements, usage of company name, changes markdown files to rst format, and removes "MapReduce Jobs" wherever used.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ programs and tools.
 
 `Twitter-pack: </cdap-twitter-pack/>`_ Twitter Integration
   The Twitter-pack library is designed to ease the CDAP apps development tasks that require integration with Twitter, 
-  such as pulling tweets stream.
+  such as pulling a tweets stream.
```
Comment (jwang47) [inline README.rst]: "pulling a tweets stream" is kind of ambiguous to me. Maybe "pulling from a stream of tweets"?

Comment (jwang47) [inline README.rst]: Or maybe "pulling from a public stream of tweets"

File: cdap-etl-pack/README.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Batch and Realtime ETL
 ----------------------
 
 Currently, the *ETL-pack* supports two types of ETL pipelines: batch and real-time. Batch processing happens with help of 
-MapReduce jobs. Real-time processing utilizes a Flow. There are a number of configuration options 
+MapReduce programs. Real-time processing utilizes a Flow. There are a number of configuration options
```
Comment (jwang47) [inline cdap-etl-pack/README.rst]: Well, if we use "MapReduce programs" here, then we should also do so everywhere else.

**Comment by jwang47** [comment]
Two comments. Otherwise LGTM.

File: cdap-etl-pack/README.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-etl-pack/README.rst]: MapReduce programs it is!

**Pull request merged**